### PR TITLE
Building windows

### PR DESCRIPTION
**Pull request by rosaguilar**
Goal:
To describe how to build locally the QGIS documentation (html) in Windows. 
The idea is to test this minimum setting to guide Windows users and hopefully gain more documenters

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: readme.rst [hunk 1]
```diff
@@ -304,6 +304,53 @@ General use:
...
+- Pip (https://pip.pypa.io/en/stable/installing/)
+- Virtualenv (https://virtualenv.pypa.io/en/latest/installation/)
+
+#. Create a local copy of **your** QGIS doc repository.
+ 
+   .. code-block:: bash
+
+     $ git clone https://github.com/<YourName>/QGIS-Documentation.git
+
+#. Create a virtual environment, e.g., *venv* in the folder just created *QGIS-Documentation*.
```
Comment (havatv) [inline readme.rst]:
```suggestion
#. Create a virtual environment, e.g., *venv* in the folder just created
   (*QGIS-Documentation*).
```

File: readme.rst [hunk 2]
```diff
@@ -304,6 +304,53 @@ General use:
...
+
+   .. code-block:: bash
+
+     $ Virtualenv venv
+  
+#. Activate the virtual environment. 
+   On Windows, virtualenv creates a batch file that can be located at:
+   
+   ..\\QGIS-Documentation\\venv\\Scripts\\activate.bat.
```
Comment (havatv) [inline readme.rst]:
```suggestion
   venv\\Scripts\\activate.bat. 
```

File: readme.rst [hunk 3]
```diff
@@ -304,6 +304,53 @@ General use:
...
+ 
+   .. code-block:: bash
+
+     $ git clone https://github.com/<YourName>/QGIS-Documentation.git
+
+#. Create a virtual environment, e.g., *venv* in the folder just created *QGIS-Documentation*.
+
+   .. code-block:: bash
+
+     $ Virtualenv venv
```
Comment (havatv) [inline readme.rst]:
```suggestion
     $ cd QGIS-Documentation
     $ Virtualenv venv
```

Comment (havatv) [reply readme.rst]: For completeness...

File: readme.rst (hunk 2 above)
Comment (havatv) [reply readme.rst]: Should not be necessary to "go up and then down again".

File: readme.rst (hunk 3 above)
Comment (rosaguilar) [reply readme.rst]: Thanks, @SrNetoChan and @havatv. I added the suggestions.
Although I tested myself, It would be nice if someone else can test it.

**Comment by havatv** [comment]
Should we merge? I someone tests on Windows and discovers a problem, we can fix that in a later PR.

**Comment by SrNetoChan** [comment]
I agree. Let's merge it. I will try to try it later.

**Comment by havatv** [comment]
Thanks, @rosaguilar!

**Pull request merged**